utils/index.js

```diff
@@ -12,3 +14,18 @@ export function formatPetsAge(seconds) {
   parts.push(`${seconds}s`);
   return parts.join(" ");
 }
+
+export function SVGIcon({ variant, color = "black", size = 50, ariaLabel }) {
```

Comment:
this is not a utils function, it is a react function component and should be treated as it